### ISSUE TITLE
Extra information

### ISSUE THREAD

**Issue by Abdou741** [mlrabbitt/avaya-sip#1]
Hi,
If I understood correctly, you have installed a VOIP server and between there is a gateway (proxy) able to make the connection between the Avaya telephone set and the server? Do you have a server recommendation to use?

**Comment by mlrabbitt** [comment]
I have been using this proxy script in front of a 3CX PBX although I would not suggest trying to make it work unless you have a small number of phones that you have easy access to.  One of the problems with running Avaya phones in SIP firmware is that if you have network disruption or need to update or restart your PBX and the phones can't REGISTER for a certain period of time, they will just revert back to the login screen and will need to be logged in manually again.

**Comment by Abdou741** [comment]
Yes it is really on an experimental basis for the moment it is a 3CX on a VPS (or local) and only one avaya station which would be connected to it. I will try to reproduce it. Thank you !
